### PR TITLE
Proper photos, missing content, live data

### PR DESCRIPTION
**Pull request by DanielBrierton**

### CODE REVIEW

**Comment by DanielBrierton** [comment]
Deploy preview for *cdf-website* ready!

Built with commit 1e4e3bde6832870264d44072e8e62dc0dfbc5d7e

https://deploy-preview-4--cdf-website.netlify.com

File: components/index/Community.vue [hunk 1]
```diff
@@ -7,21 +7,21 @@
...
-          <img src="https://picsum.photos/285/175?image=2" />
+          <img src="~/static/images/index/megadojo.jpg" />
           <div class="c-community__card-text">
             <h3>MegaDojos</h3>
             <p>{{ $t('Lorem ipsum dolor sit amet, nec ex timeam prompta invenire, no mea') }}</p>
           </div>
         </a>
         <a href="#" class="c-community__card">
-          <img src="https://picsum.photos/285/175?image=3" />
+          <img src="~/static/images/index/dojocon.jpg" />
```
Comment (heyitsmaimai) [inline components/index/Community.vue]: picture quality looks a bit funky?

File: components/index/Volunteer.vue [hunk 2]
```diff
@@ -45,10 +52,35 @@
         text-align: left;
       }
 
-      p {
+      > p {
```
Comment (heyitsmaimai) [inline components/index/Volunteer.vue]: whats the < for?

Comment (Wardormeur) [reply components/index/Volunteer.vue]: It means "direct child", as only direct children will be affected rather than any recursive children

**Comment by conoramurphy** [comment]
@aguycalledconor needs to add the proper picture links in the ticket here to move this on.

**Comment by Wardormeur** [comment]
I'll set the PR for Hero images apart, due to the lenght of this one already

**Pull request merged**